tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot configurationRoot;
+        public PermitServiceApiConfiguration permitServiceConfig = new();
```

Comment:
This should be called PermitServiceConfig if it is public.

Reply:
Hi John
Thanks for you comment. As permitServiceConfig is a class object. Therefore we have used camelCase naming convention.

Reply:
Hi, that's not correct - Pascal casing please.

Reply:
Please also provide public get, private set?

Reply:
Hey John
We have made both the changes as per the review comments.